tests/_utils/run_sql

```diff
@@ -2,11 +2,17 @@
 # parameter 1: sql
 # parameter 2: database host
 # parameter 3: database port
+# parameter 4: other mysql client settings
 
 set -e
 
-host=${2:-127.0.0.1}
-port=${3:-4000}
+sql=${1}
+shift
```

Comment:
The failure test case may be caused by the shift operation.
some `run_sql sql_statement` doesn't provide `host` and `port`